WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/BackupDownloadActivity.kt

```diff
@@ -0,0 +1,64 @@
+package org.wordpress.android.ui.jetpack.backup
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import kotlinx.android.synthetic.main.backup_download_activity.*
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.ui.LocaleAwareActivity
+import org.wordpress.android.ui.jetpack.backup.details.BackupDownloadDetailsFragment
+import javax.inject.Inject
+
+class BackupDownloadActivity : LocaleAwareActivity() {
+    @Inject internal lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var viewModel: BackupDownloadViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        (application as WordPress).component().inject(this)
+        setContentView(R.layout.backup_download_activity)
+
+        setSupportActionBar(toolbar_main)
+        supportActionBar?.let {
+            it.setHomeButtonEnabled(true)
+            it.setDisplayHomeAsUpEnabled(true)
+            // todo: annmarie - this is here to get past lint - this will be set from childfrag
+            it.title = getString(R.string.backup_download_details_page_title)
+        }
+        setupViewModel()
+    }
+
+    private fun setupViewModel() {
+        viewModel = ViewModelProviders.of(this, viewModelFactory)
+                .get(BackupDownloadViewModel::class.java)
+        viewModel.start()
+
+        // todo: annmarie temporary start
+        val fragment = BackupDownloadDetailsFragment.newInstance()
```

Comment:
I'm not sure if this is why you added the todo, but I thought I'd better comment so we don't miss it -> this line will create a new instance of the fragment even after rotation.

If that's why you added the todo, feel free to ignore this comment.

Reply:
@malinajirka - Oh yeah, this is just so I can force a start - it is not the final code. :)